index.js

```diff
@@ -59,26 +59,49 @@ class SteamAuth {
         const response = await axios.get(
           `https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v0002/?key=${this.apiKey}&steamids=${steamId}`
         );
+
         const players =
           response.data &&
           response.data.response &&
           response.data.response.players;
 
-        if (players && players.length > 0) {
+        if (players && players.length > 0)
+        {
```

Comment:
Nit: The brace style here doesn't match the rest of the codebase. Use `if (condition) { ... }` to keep it consistent.

```suggestion
        if (players && players.length > 0) {
```